README.md

```diff
@@ -28,31 +28,134 @@ flag is highly recommended for those new to [pip](https://pip.pypa.io).
+image_ids = ['3949357_1454705_2020-12-01_241c']
+order_detail = OrderDetails(
+    'test_order',
+    [Product(image_ids, 'analytic', 'psorthotile')]
+)
 
-Using `planet init` your account credentials (login/password) can be used to obtain the api key.
+async def create_order(order_detail):
+    async with APlanetSession(auth=(API_KEY, '')) as ps:
+        client = AOrdersClient(ps)
+        return await client.create_order(order_detail)
 
+oid = asyncio.run(create_order())
+print(oid)
+```
+
+Not into async? No problem. Just wrap the library and async operations together
+and call from your synchronous code.
 
-# Example CLI Usage
+```python
+def sync_create_order():
+    return asyncio.run(create_order())
+
+print(sync_create_order)
+```
+
+Why async? Because things get *really cool* when you want to work with multiple
+orders. Here's an example of submitting two orders, waiting for them to
+complete, and downloading them. The orders each clip a set of images to a
+specific area of interest (AOI), so they cannot be combined into one order.
+(hint: [Planet Explorer](https://www.planet.com/explorer/)) was used to define
+the AOIs and get the image ids.
+ 
+
+```python
+import asyncio
+import os
+from planet.api import APlanetSession, AOrdersClient
+from planet.api.order_details import OrderDetails, Product, Tool
+
+API_KEY = os.getenv('PL_API_KEY')
+
+iowa_aoi = {
+    "type": "Polygon",
+    "coordinates": [[
+        [-91.198465, 42.893071],
+        [-91.121931, 42.893071],
+        [-91.121931, 42.946205],
+        [-91.198465, 42.946205],
+        [-91.198465, 42.893071]]]
+}
+
+iowa_images = [
+    '20200925_161029_69_2223',
+    '20200925_161027_48_2223'
+]
+iowa_order = OrderDetails(
+    'iowa_order',
+    [Product(iowa_images, 'analytic', 'PSScene4Band')],
+    tools=[Tool('clip', {'aoi': iowa_aoi})]
+)
+
+oregon_aoi = {
+    "type": "Polygon",
+    "coordinates": [[
+        [-117.558734, 45.229745],
+        [-117.452447, 45.229745],
+        [-117.452447, 45.301865],
+        [-117.558734, 45.301865],
+        [-117.558734, 45.229745]]]
+}
+
+oregon_images = [
+    '20200909_182525_1014',
+    '20200909_182524_1014'
+]
+oregon_order = OrderDetails(
+    'oregon_order',
+    [Product(oregon_images, 'analytic', 'PSScene4Band')],
+    tools=[Tool('clip', {'aoi': oregon_aoi})]
+)
+
+
+async def create_and_download(order_detail, client):
+    oid = await client.create_order(order_detail)
+    print(oid)
+    state = await client.poll(oid, verbose=True)
+    print(state)
+    filenames = await client.download_order(oid, progress_bar=True)
+    print(f'downloaded {oid}, {len(filenames)} files downloaded.')
+
+
+async def main():
+    async with APlanetSession(auth=(API_KEY, '')) as ps:
+        client = AOrdersClient(ps)
+        await asyncio.gather(
+            create_and_download(iowa_order, client),
+            create_and_download(oregon_order, client)
+        )
+
+asyncio.run(main())
```

Comment:
This specific example felt opaque on the progress. Consider adding a timestamp to the status messages or even a fancy "running for <n> <units-of-time>" that gets updated in-place. Also consider changing the order details for this example so it doesn't take several minutes to complete. :)

Reply:
Great idea re: timestamp. And I agree it would be great to have the code example complete faster here. I do not have any insight into ways to decrease order completion time via the order details. I could try by removing the clip tool, though that would make the example a bit less interesting because there is no reason to use two separate orders if they do not have different toolchains.

Reply:
Yeah, timestamps sound good! I have a fancy in-place visual cue implemented as a part of the orders client for downloading orders. We could consider adding similar functionality for polling orders as well. If that sounds like it would make the user experience better, I can write up an issue to implement it.

Side question: for the fancy in-place terminal updates for downloads (and potentially for polling), would you expect displaying the updates to be the default behavior, disabled by a `quiet` flag or to have it be disabled by default and enabled with a `verbose` flag?

Reply:
I just realized I had assumed you were just talking about the polling portion of this code. Was I correct in that or is there a way the download portion could be improved as well?

Reply:
I am going to just break this off as a different effort, in-place updates with status and duration: #253

Reply:
Argh, I think the internet swallowed my reply. :) For the inline polling, I think the timestamp could be helpful for an example, because that first order can take quite a while, especially for a user that isn't expected it. With regard to CLI functionality only, I personally would expect the default to show indicators and then get silenced with a `--quiet` or `--silent` or something tantamount. :)